frontend/templates/admin/dashboard/overview.html

```diff
@@ -0,0 +1,292 @@
+        ></div>
+      </div>
+    </div>
+  </div>
+  <script src="https://unpkg.com/chartist"></script>
+  <script src="https://unpkg.com/chartist-plugin-axistitle"></script>
+  <script type="module">
+    import { LineChart } from "https://unpkg.com/chartist@1.3.1/dist/index.js";
+
+    const tooltipElement = document.createElement("div");
+    tooltipElement.className = "chartist-tooltip hidden";
+    document.querySelector(".ct-chart").appendChild(tooltipElement);
+
+    // TODO: make AJAX call to ticketing/revenue API to
+    // get current revenue for current month
+
+    const aprilRevenueData = [
+      { meta: "Apr 1, 2025", value: 4750.25 },
+      { meta: "Apr 2, 2025", value: 5125.8 },
+      { meta: "Apr 3, 2025", value: 4980.35 },
+      { meta: "Apr 4, 2025", value: 5610.9 },
+      { meta: "Apr 5, 2025", value: 3890.45 },
+      { meta: "Apr 6, 2025", value: 3250.75 },
+      { meta: "Apr 7, 2025", value: 4520.3 },
+      { meta: "Apr 8, 2025", value: 5830.65 },
+      { meta: "Apr 9, 2025", value: 6125.2 },
+      { meta: "Apr 10, 2025", value: 5740.85 },
+      { meta: "Apr 11, 2025", value: 6310.5 },
+      { meta: "Apr 12, 2025", value: 4890.35 },
+      { meta: "Apr 13, 2025", value: 3780.4 },
+      { meta: "Apr 14, 2025", value: 5240.95 },
+      { meta: "Apr 15, 2025", value: 6450.3 },
+      { meta: "Apr 16, 2025", value: 5820.75 },
+      { meta: "Apr 17, 2025", value: 5680.2 },
+      { meta: "Apr 18, 2025", value: 6240.85 },
+      { meta: "Apr 19, 2025", value: 4930.4 },
+      { meta: "Apr 20, 2025", value: 3760.9 },
+      { meta: "Apr 21, 2025", value: 5340.25 },
+      { meta: "Apr 22, 2025", value: 6540.8 },
+      { meta: "Apr 23, 2025", value: 6190.35 },
+      { meta: "Apr 24, 2025", value: 5870.7 },
+      { meta: "Apr 25, 2025", value: 6420.15 },
+      { meta: "Apr 26, 2025", value: 5230.5 },
+      { meta: "Apr 27, 2025", value: 3950.85 },
+      { meta: "Apr 28, 2025", value: 5480.3 },
+      { meta: "Apr 29, 2025", value: 6730.75 },
+      { meta: "Apr 30, 2025", value: 7120.2 },
+    ];
+
+    const dayNumberLabels = aprilRevenueData.map((item) => {
+      const dateParts = item.meta.split(" ");
+      return dateParts[1].replace(",", ""); // Remove the comma after the day
+    });
+
+    const chart = new LineChart(
+      "#chart1",
+      {
+        labels: dayNumberLabels,
+        series: [aprilRevenueData],
+      },
+      {
+        fullWidth: true,
+        width: "100%",
+        height: "100%",
+        showArea: true,
+        low: 0,
+        chartPadding: {
+          bottom: 20,
+          left: 40,
+        },
+        plugins: [
+          Chartist.plugins.ctAxisTitle({
+            axisX: {
+              axisTitle: "Days of April",
+              axisClass: "ct-axis-title",
+              offset: {
+                x: 0,
+                y: 40,
+              },
+              textAnchor: "middle",
+            },
+            axisY: {
+              axisTitle: "Revenue ($)",
```

Comment:
This sideways text is kind of hard to read, not sure the best way of improving it though. Maybe set text-orientation to upright or just sacrifice some horizontal real estate to display it horizontally?